controllers/extension-dns-service/charts/internal/dns-service/values.yaml

```
@@ -0,0 +1,23 @@
+# Default values for dns-service in seed.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicas: 1
+
+images:
+  dns-controller-manager: eu.gcr.io/gardener-project/dns-controller-manager:0.6.2-master
```

Don't hard-code the image here. Instead, put it into the image vector and fetch it from there.
```suggestion
  dns-controller-manager: repo:tag
```